src/lightning_lite/utilities/imports.py

```diff
@@ -0,0 +1,163 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
This was all copied as needed. The package utils will eventually be replaced by the lightning-tools repo and the rest of the constants shouldn't be a big concern for duplication. Some of them stay in PL, most of them move here. 
Hence, not a clean file move.

Reply:
fyi: Once it's on PyPI this should work #14475

Reply:
Integrated it here too now